loadusc/loaduscxcite.py

```diff
@@ -9,92 +9,148 @@
     with open(releasepointJSONPath, 'r') as f:
         releasepoints = json.load(f)
     releasepoints_rev = releasepoints[::-1]
-    
+
     # pljson is a list of public laws and their enactment dates
     with open(publawsDict, 'r') as f:
         pljson = json_util.loads(f.read())
-    
-    pls = [item.get('name') for item in releasepoints_rev if not (re.search(r'not',item.get('name')) or re.search(r'u1$',item.get('name')))]
-    plsDict = dict((item.get('name'), index) for index, item in enumerate(releasepoints_rev))
+
+    pls = [
+        item.get('name')
+        for item in releasepoints_rev
+        if not (
+            re.search(r'not', item.get('name')) or re.search(r'u1$', item.get('name'))
+        )
+    ]
+    plsDict = dict(
+        (item.get('name'), index) for index, item in enumerate(releasepoints_rev)
+    )
     print(plsDict)
     pls.sort(key=sortPLS)
     prior_index = 0
     for pl in pls:
-        pljsonitem = pljson.get(pl) 
+        pljsonitem = pljson.get(pl)
         if not pljsonitem:
             logger.error('No item for ' + str(pl))
             continue
         publaw_date = pljsonitem.get('publawDate')
         if publaw_date:
             release_date = publaw_date.strftime('%m/%d/%Y')
         # Get the list of names from the last pl until and including the current one
-        plIndex=plsDict.get(pl)
-        for rp in releasepoints_rev[prior_index:plIndex+1]:
+        plIndex = plsDict.get(pl)
+        for rp in releasepoints_rev[prior_index : plIndex + 1]:
             rpname = rp.get('name')
             release_point_path = os.path.join(USC_RELEASEPOINT_DIRPATH, rpname)
             logger.info(release_point_path)
             dbload = None
             try:
-                logger.info('Loading release point ' + rpname + ' for date: ' + release_date)
-                logger.info(str([XCITEDBPATH, '-db', XMLDBPATH, '-date', release_date, 'load-xml', '-r', release_point_path]))
-                dbload = subprocess.run([XCITEDBPATH, '-db', XMLDBPATH, '-dc', DOCCONFIGPATH, '-date', release_date, 'load-xml', '-r', release_point_path] , timeout=600, capture_output=True)
+                logger.info(
+                    'Loading release point ' + rpname + ' for date: ' + release_date
+                )
+                logger.info(
+                    str(
+                        [
+                            XCITEDBPATH,
+                            '-db',
+                            XMLDBPATH,
+                            '-date',
+                            release_date,
+                            'load-xml',
+                            '-r',
+                            release_point_path,
+                        ]
+                    )
+                )
+                dbload = subprocess.run(
+                    [
+                        XCITEDBPATH,
```

Comment:
won't it fail if the folder does not exist?

maybe it is worth checking somewhere around `constants.py` that folders exist and are accessible?

Reply:
We now getting it from ENV vars, if they not setup properly the default value will be used, if it fails also it is normal to fail here.